src/progress.rs

```diff
@@ -1333,7 +1333,9 @@ mod tests {
     fn late_pb_drop() {
         let pb = ProgressBar::new(10);
         let mpb = MultiProgress::new();
-        mpb.add(pb);
+        // This clone call is required to trigger a now fixed bug.
```

Comment:
Can you add a suppression of the clippy lint?

Reply:
I think it's now suppressed, but I couldn't make clippy emit the warning so I'm not sure. Can you share how you run clippy to make the warning appear?